### ISSUE TITLE
Unable to import image dataset with yolo annotations

### ISSUE THREAD

**Issue by bit-scientist** [opencv/cvat#5653]
I want to import my dataset into CVAT and refine existing annotations. To achieve this I'm trying to import my dataset (images and labels in yolo format) via CVAT's **Import dataset** located in **Actions**. I am not aware if it is doable, but I did the following anyways.

- [x] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Steps to Reproduce (for bugs)

Not sure if it is a bug or I'm just not doing it right.
### Expected Behaviour

It should import the dataset and show annotations on each image respectively.
### Current Behaviour

It throws an error shown below
![image](https://user-images.githubusercontent.com/26497571/215987518-3f343cdd-1f83-4629-b263-4046e7639740.png)

### Possible Solution

No idea how to solve
### Context

1. there are jpg images and their corresponding txt files in yolo format
1. images and annotations reside in `obj_train_data` folder
1. `obj.data` file contains the following:
    ```
    classes = 50
    train = train.txt
    names = obj.names
    backup = backup/
    ```
1. `obj.names` contains 50 class names on a row basis
2. `train.txt` contains the paths of jpg images in  `obj_train_data` as follows:
    ```
    obj_train_data\8947_서울3720.jpg 
    obj_train_data\2742_서울5789.jpg 
    obj_train_data\7626_FAIL.jpg 
    ```
In short, the structure is exactly the same  as defined in [YOLO export](https://opencv.github.io/cvat/docs/manual/advanced/formats/format-yolo/#yolo-export).

I hope someone can help me to handle this. Thank you!

### Your Environment

- Docker version: `v4.16.3`:
- Are you using Docker Swarm or Kubernetes? **No**
- Operating System and version (e.g. Linux, Windows, MacOS): **Windows 10, x64**
- Code example or link to GitHub repo or gist to reproduce problem: **Not available**
- Other diagnostic information / logs: **Not available**
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

**Comment by bit-scientist** [comment]
hi, @yasakova-anastasia, may I have your attention here please. I saw you answering to another similar question and thought you could be of help. Thank you

**Comment by bit-scientist** [comment]
I don't know why this issue isn't being addressed at all. Is it improperly defined or something else I am unaware of? @zhiltsov-max,  @klakhov could you guys guide me here? Thank you.

**Comment by zhiltsov-max** [comment]
Hi, please try to save the text files in the UTF-8 without BOM encoding.

**Comment by bit-scientist** [comment]
Thank you for your response. I checked the txt files' encodings with Notepadd++ Encoding panel, it's already **UTF-8**.

**Comment by zhiltsov-max** [comment]
Could you please try to open the .txt files in the dataset by Python? 

Create a file like this:

```python
from glob import glob
import sys

dataset_root = sys.argv[1]

for filename in glob(dataset_root + '/**/*.txt', recursive=True):
    print(f"testing '{filename}'...")
    with open(filename) as f:
        f.read()
```

And run it in the terminal passing the dataset root directory path:

`python ./check_files.py <dataset_root_path>`

**Comment by bit-scientist** [comment]
Thank you, turns out the image txt files are all **UTF-8** encoded, but `train.txt` which I created  (using Powershell's `Get-ChildItem -Path $PWD -Filter *.jpg | Select-Object -ExpandProperty FullName | Out-File -FilePath "$PWD\train.txt"`) was **UTF-16 LE BOM**. So I changed it to **UTF-8** and the issue has gone. Thanks once again for your guidance.
